shared/profile/showcase-team-offer/index.js

```diff
@@ -0,0 +1,159 @@
+// @flow
+import * as React from 'react'
+import {globalColors, globalMargins, globalStyles, isMobile} from '../../styles'
+import {
+  Avatar,
+  Box,
+  Button,
+  ClickableBox,
+  Divider,
+  Icon,
+  Meta,
+  PopupDialog,
+  ScrollView,
+  Text,
+} from '../../common-adapters'
+import {teamWaitingKey} from '../../constants/teams'
+
+import type {RowProps, Props} from './index'
+
+const TeamRow = ({canShowcase, name, isOpen, membercount, onPromote, showcased, waiting}: RowProps) => (
+  <Box style={globalStyles.flexBoxColumn}>
+    <Box
+      key={name}
+      style={{
+        ...globalStyles.flexBoxRow,
+        minHeight: isMobile ? 64 : 48,
+        marginRight: globalMargins.small,
+        paddingTop: globalMargins.tiny,
+        paddingBottom: globalMargins.tiny,
+      }}
+    >
+      <Box style={{display: 'flex', position: 'relative'}}>
+        <Avatar
+          isTeam={true}
+          size={isMobile ? 48 : 32}
+          style={{marginLeft: globalMargins.tiny}}
+          teamname={name}
+        />
+      </Box>
+      <Box style={{...globalStyles.flexBoxColumn, flex: 1, marginLeft: globalMargins.small}}>
+        <Box style={globalStyles.flexBoxRow}>
+          <Text type="BodySemibold">{name}</Text>
+          {isOpen && <Meta title="OPEN" style={styleMeta} />}
+        </Box>
+        <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+          <Text type="BodySmall">{membercount + ' member' + (membercount !== 1 ? 's' : '')}</Text>
+        </Box>
+      </Box>
+      {showcased ? (
+        <Button
```

Comment:
in this case we should likely just have one button and use showcased to switch the label / onClick param etc